src/main/java/org/sagebionetworks/web/client/widget/entity/download/UploadDialogWidgetV2.java

```diff
@@ -27,7 +26,7 @@ public class UploadDialogWidgetV2 extends Widget {
 
   @Inject
   public UploadDialogWidgetV2(
-    GlobalApplicationStateImpl globalApplicationState,
```

Comment:
This erroneous injection registered the uploader's drag & drop handler on a new instance of GlobalApplicationStateImpl instead of the singleton.